src/server/game/Maps/MapMgr.cpp

```diff
@@ -230,7 +230,7 @@ Map::EnterState MapMgr::PlayerCannotEnter(uint32 mapid, Player* player, bool log
             instaceIdToCheck = save->GetInstanceId();
 
         // instaceIdToCheck can be 0 if save not found - means no bind so the instance is new
-        if (!player->CheckInstanceCount(instaceIdToCheck))
+        if (!player->CheckInstanceCount(instaceIdToCheck) && !entry->IsRaid())
```

Comment:
how about using `enty->IsNonRaidDungeon()` instead of `entry->isDungeon()` right above to not enter this whole block for raids